internal/api/handlers/operations_handler.go

```diff
@@ -130,6 +130,27 @@ func (h *OperationsHandler) CancelOperation(ctx context.Context, request *api.Ca
 	return &api.CancelOperationResponse{}, nil
 }
 
+func (h *OperationsHandler) GetOperation(ctx context.Context, request *api.GetOperationRequest) (*api.GetOperationResponse, error) {
+	handlerLogger := h.logger.With(zap.String(logs.LogFieldSchedulerName, request.GetSchedulerName()), zap.String(logs.LogFieldOperationID, request.GetOperationId()))
+	handlerLogger.Info("received request to get operation by id")
+	op, _, err := h.operationManager.GetOperation(ctx, request.GetSchedulerName(), request.GetOperationId())
+	if err != nil {
+		if errors.Is(err, portsErrors.ErrNotFound) {
+			handlerLogger.Warn("operation not found", zap.String(logs.LogFieldSchedulerName, request.GetSchedulerName()), zap.String(logs.LogFieldOperationID, request.GetOperationId()), zap.Error(err))
```

Comment:
```suggestion
			handlerLogger.Warn("operation not found", zap.String(logs.LogFieldSchedulerName, request.GetSchedulerName()), zap.Error(err))
```
We already defined the op ID label at line 134